pkg/sriov/sriov.go

```diff
@@ -277,6 +279,11 @@ func (s *sriovManager) ApplyVFConfig(conf *sriovtypes.NetConf) error {
 		return fmt.Errorf("failed to lookup master %q: %v", conf.Master, err)
 	}
 
+	origVfInfo := getVfInfo(pfLink, conf.VFID)
```

Comment:
We should try save and restore the original VF states from this. And on the ApplyVFConfig() should only try setting VF configurations that are given in the config. If any config not given let's not touch them.

Reply:
If we can define a data structure for origVfInfo, then it would be easier to store the retrieved origVfInfo to the defined structure and cache it. This will save us from contional check from [here](https://github.com/intel/sriov-cni/pull/114/files#diff-73126a8b935928a81f24ce95cd923a2dR341)